cmd/katapult/config_test.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/krystal/katapult-cli/config"
+	"github.com/stretchr/testify/assert"
+)
+
+const mockConfigFormat = `---
+api_key: %s
+api_url: %s
+
+`
+
+func TestConfig(t *testing.T) {
+	tests := []struct {
+		name string
+
+		apiKey string
+		apiURL string
+	}{
+		{
+			name:   "empty values",
+			apiKey: "",
+			apiURL: "",
+		},
+		{
+			name:   "only API URL blank",
+			apiKey: "test",
+			apiURL: "",
+		},
+		{
+			name:   "both fields present",
+			apiKey: "test",
+			apiURL: "test",
+		},
+		{
+			name:   "only API key blank",
+			apiKey: "",
+			apiURL: "test",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(fmt.Sprintf("key: %s, url: %s", tt.apiKey, tt.apiURL), func(t *testing.T) {
+			conf, err := config.New()
+			assert.NoError(t, err)
+			conf.SetDefault("api_key", tt.apiKey)
+			conf.SetDefault("api_url", tt.apiURL)
+			cmd := configCommand(conf)
+			out := &bytes.Buffer{}
+			cmd.SetOut(out)
+			if err := cmd.RunE(cmd, []string{}); err != nil {
+				t.Fatal(err)
+			}
```

Comment:
I'd suggest replacing this a `require.NoError(...)`.

Reply:
Actually, any reason this test can't or shouldn't use the `testAssertCommand` helper function?